### PR TITLE
atom-editor: version bump v1.11.2

### PR DESCRIPTION
**Pull request by t-richards**

### CODE REVIEW

**Comment by sjug** [comment]
Thanks for the PR @t-richards, is there a new build strategy for atom?

**Comment by samueloph** [comment]
@sjug Yes there is, that's why i didn't updated yet.

I'm gonna check @t-richards PR right now.

**Comment by t-richards** [comment]
@sjug The build strategy is largely the same, but it looks like the install script only allows installation to a hardcoded path of `/usr/local`. This is why I changed the `package()` function so significantly.

File: PKGBUILD [hunk 1]
```diff
@@ -37,15 +37,36 @@ build() {
   cd "$srcdir/atom-$pkgver"
 
   export PYTHON=python2
-  script/build --build-dir "$srcdir/atom-build"
+  script/build --compress-artifacts
```
Comment (samueloph) [inline PKGBUILD]: What's the motivation behind this? At first glance i see no reason to compress the binary, as we are not using the generated tarball.

**Review by samueloph — CHANGES_REQUESTED** [review]
Is --compress-artifacts really needed /or useful ?

File: PKGBUILD (hunk 1 above)
Comment (t-richards) [reply PKGBUILD]: You're correct; I was thinking this flag might have some impact on the compiled `atom` binary size (it does not, in fact, have any such effect).

**Review by samueloph — APPROVED** [review]

**Comment by samueloph** [comment]
Ok, everything's looking good now. I will wait a few hours to see if @sjug has something to say, if not, i will merge and upload it to AUR.

**Comment by sjug** [comment]
I trust you @samueloph thanks guys for putting this together, lets push it up to the AUR.

**Pull request merged**